content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -374,6 +374,12 @@ const prisma = new PrismaClient({
 
 ### <inlinecode>rejectOnNotFound</inlinecode>
 
+<Admonition type="info">
+
+**Deprecated:** `rejectOnNotFound` is deprecated in v4.0.0. From v4.0.0, use the queries [`findUniqueOrThrow`](#inlinecodefinduniqueorthrowinlinecode) or [`findFirstOrThrow`](#inlinecodefindfirstorthrowinlinecode).
```

Comment:
```suggestion
**Deprecated:** `rejectOnNotFound` is deprecated in v4.0.0. From v4.0.0, use the methods [`findUniqueOrThrow`](#inlinecodefinduniqueorthrowinlinecode) or [`findFirstOrThrow`](#inlinecodefindfirstorthrowinlinecode).
```

Or do we normally call them "queries" throughout the docs?

Reply:
We might not always be consistent, unfortunately, but we call them model queries in this doc and queries elsewhere in the docs, so I'll go with this.